tutorial/Lab2/README.md

```diff
@@ -0,0 +1,151 @@
+# Lab 2. I need to change but none of the hassle
```

Comment:
```# Lab 2. Updating your app to pick up chart changes```

Reply:
I am going to stick with the titles if that is ok with you?  I think they add some fun to the tutorial.

Reply:
Ok, done